src/main/java/io/debezium/connector/spanner/db/metadata/SchemaRegistry.java

```diff
@@ -100,7 +114,30 @@ public synchronized boolean updateSchema(Timestamp updatedTimestamp) {
         return true;
     }
 
-    private void forceUpdateSchema(Timestamp updatedTimestamp) {
+    public void setSpannerSchema(SpannerSchema spannerSchema) {
```

Comment:
If this is a test-only function, we should annotate it that way, also in the comments